packages/style/themes/default/_theme.template.scss

```diff
@@ -2,6 +2,7 @@
 
 // QueryBuilder
 $query-builder-bg: $white;
+$query-builder-border-color: #D8E1EB;
```

Comment:
should our colors refer to a theme define variable, even for the template?

Reply:
we could if we have repetitive colors in the template